tests/bsim/bluetooth/tests.nrf5340bsim_nrf5340_cpunet.txt

```diff
@@ -2,3 +2,16 @@
 # built in the net core)
 # This file is used in CI to select which tests are run
 tests/bsim/bluetooth/ll/
+tests/bsim/bluetooth/host/att/eatt_notif
+tests/bsim/bluetooth/host/misc/disable
+tests/bsim/bluetooth/host/misc/unregister_conn_cb
+tests/bsim/bluetooth/host/adv/periodic
+tests/bsim/bluetooth/host/adv/extended
+tests/bsim/bluetooth/host/adv/chain
+tests/bsim/bluetooth/host/l2cap/send_on_connect
+tests/bsim/bluetooth/host/central
+tests/bsim/bluetooth/host/privacy/peripheral
+tests/bsim/bluetooth/host/privacy/central
+tests/bsim/bluetooth/host/gatt/authorization
+tests/bsim/bluetooth/host/gatt/general
```

Comment:
What do we gain from running these on the cpunet?
I think it's OK to add one or two just to do a smoke test, but I'm not sure we would see any significant change in behavior between the nrf52bsim and nrf5340 cpunet, and these would probably have been better suited for the cpuapp where there's a more noticeable difference in behavior.